compiler/z/codegen/OMRTreeEvaluator.cpp

```diff
@@ -14116,6 +14116,148 @@ bool isSettingOp(uint8_t byteValue, TR::InstOpCode::Mnemonic SI_opcode)
       return false;
    }
 
+TR::Register *
+OMR::Z::TreeEvaluator::bitpermuteEvaluator(TR::Node *node, TR::CodeGenerator *cg)
+   {
+   TR::Node *value = node->getChild(0);
+   TR::Node *addr = node->getChild(1);
+   TR::Node *length = node->getChild(2);
+
+   bool nodeIs64Bit = node->getSize() == 8;
```

Comment:
This variable is unused. Can we remove?

Reply:
Addressed in 537a564e0b4cc5d2b99a94cb69e8ef62d0a349d3